sunpy/net/tests/test_fido.py

```diff
@@ -492,6 +492,14 @@ def test_fido_no_time(mocker):
     jsoc_mock.assert_called_once()
 
 
+@pytest.mark.remote_data
+def test_jsoc_missing_email():
+    res = Fido.search(a.Time("2011/01/01", "2011/01/01 00:01"), a.jsoc.Series.aia_lev1_euv_12s)
+
+    with pytest.raises(ValueError, match="Pass attrs.jsoc.Notify with a JSOC registered email as an argument to fido.search"):
```

Comment:
```suggestion
    with pytest.raises(ValueError, match=r"A registered email is required to get data from JSOC.*"):
```
This is going to go badly.